### PR TITLE
aspell-el added

### PR DESCRIPTION
**Pull request by nereusx**
Be polite, it is my first try ;)

### CODE REVIEW

**Comment by flexibeast** [comment]
Thanks for your contribution!

Please run `xlint` on `template` files before submitting them. In this instance, `xlint` [reports](https://travis-ci.org/void-linux/void-packages/jobs/606276321?utm_medium=notification&utm_source=github_status):

* the `depends=` line needs to come after the `hostmakedepends=` line;
* the value for `revision=` must not be zero - it should be `1`.

When you update your PR, please make sure to commit your changes via `git commit --amend`, with a commit message of the form `New package: <pkgname>-<version>`, and then force-push to `nereusx:master`.

File: srcpkgs/aspell-el/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'aspell-el'
+#
+#	This is copy of aspell's site
+#	https://ftp.gnu.org/gnu/aspell/dict/0index.html
+#
+#	Project's page: http://www.elspell.gr/
```
Comment (Chocimier) [inline srcpkgs/aspell-el/template]: This should be put into homepage.

File: srcpkgs/aspell-el/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#
+#	Void Linux Package:
+#	Nicholas Christopoulos <nereus@freemail.gr>
+#
+pkgname=aspell-el
+version=0.08
+revision=0
```
Comment (Chocimier) [inline srcpkgs/aspell-el/template]: Revision means which time same version of package is built in Void, so distfiles can't depend on it. For avoiding dash in version, do like in libcli template.

File: srcpkgs/aspell-el/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+archs=noarch
+wrksrc="aspell6-el-${version}-${revision}"
+build_style=configure
+depends="aspell"
+hostmakedepends="aspell-devel"
+short_desc="Greek dictionary for aspell"
+maintainer="Steve Stavropoulos <steve@math.upatras.gr>"
+license="GPL-2.0-or-later"
```
Comment (Chocimier) [inline srcpkgs/aspell-el/template]: -only

File: srcpkgs/aspell-el/template [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+pkgname=aspell-el
+version=0.08
+revision=0
+archs=noarch
+wrksrc="aspell6-el-${version}-${revision}"
+build_style=configure
+depends="aspell"
+hostmakedepends="aspell-devel"
+short_desc="Greek dictionary for aspell"
+maintainer="Steve Stavropoulos <steve@math.upatras.gr>"
```
Comment (Chocimier) [inline srcpkgs/aspell-el/template]: This field is for name of packager, rather than name of author.

**Comment by nereusx** [comment]
Thank you both of you.
I'll write a tutorial for new packagers. I have no idea where to put it... may I ll try the forum...

**Comment by flexibeast** [comment]
@nereusx: Well, are you already aware of the [CONTRIBUTING](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md), which is referenced in this repo's README? If so, do you have any suggestions for how it could be improved?

**Comment by the-maldridge** [comment]
Similarly, Manual.md is of great value.

**Comment by nereusx** [comment]
Well, it is useful, but needs work; less text more examples. You should starting with what ppl had to read already. Also add this in the must read https://github.com/iiitkottayam/first-contribution or something better. While I did already contributions on other projects like learnxiny it was simpliest, there are features and problems of git that I meet in this project since it has more traffic. You know the solutions, I ll have to search...
btw, what is xbump and where is the manual?

The Manual.md has also "missing parts" and its a bit hard to understand, for example why to use vman since I have vinstall? how to include a Makefile to original package so to do my job easy as already know, well there are more questions on this but you didn't ask.

Now, I am retired with health issues, but many years ago I was create the smallbasic mostly for PalmOS (from 2004 I retired from the project). I found cvs was more smart in simple task like add a directory without mess everything :P but that I has to say is you need a forum for each subporject (like the documentation). That worked excellent for me.

Also there is a wiki-void but link is not in the main site, why? how do you expect ppl to find this? The void site need to be rewritten with the major resources on the top. Find easy is a mesaure for sites quality you know... well at least it was.

That are my comments, I am happy with your works guys, I just comment for better results.

**Comment by flexibeast** [comment]
Thanks for your feedback!

i'm actually new to Void myself: i've only just started contributing, primarily in the area of documentation (though i've also added a package, `hunspell-en_AU`, in #13620). So i'm not one of the Void devs, and i can't comment on everything you've mentioned; but i can speak to some of the things you mentioned.

> what is xbump and where is the manual?

`xbump` is part of the `xtools` package; the man page for `xtools` says:

> xbump pkgname [git commit options] – Git commit a version bump

Could you please point to the place where `xbump` was mentioned but not explained?

> you need a forum for each subporject (like the documentation

This is already in place: see the projects listed at https://github.com/void-linux/, amongst which is the void-docs project, which has its own issue tracker.

> Also there is a wiki-void but link is not in the main site, why? how do you expect ppl to find this?

This is deliberate, as the wiki is deprecated; see [issue #104 in the void-docs repo](https://github.com/void-linux/void-docs/issues/104). i agree that whilst the wiki is still online and visible to search engines, it would be good to let people know that it should no longer be used. Myself and @bobertlo have made some suggestions about this ([here](https://github.com/void-linux/void-docs/issues/104#issuecomment-529082008) and [here](https://github.com/void-linux/void-docs/issues/104#issuecomment-546606159)), but as Void is a volunteer-run project, it relies on the relevant Void devs having the time and space to address these suggestions.

> The void site need to be rewritten with the major resources on the top.

Well, the navigation bar at the top of the Void home page has a "Documentation" entry which links to https://docs.voidlinux.org/; and, yeah, the wiki is not listed alongside it because the wiki is no longer considered an official resource.

**Comment by the-maldridge** [comment]
Also there are plenty of examples, just about 7191 as I type this.  They're conveniently located in the srcpkgs/ directory of this very repo.

**Comment by nereusx** [comment]
@flexibeast:
That is in your `contributing`: "There's a helper tool for committing packages in the xtools package: xbump"

About xbump "manual":
xbump pkgname [git commit options] – Git commit a version bump
Is it supposed to I understand it? Probably if I didn't stopped at "lower" degree in english.

Guys the most people does not know even how the mouse works. Every author of documentation has to keep this in his mind. Of course he dont need to explain the mouse, but asking to read a manual that is bad even for those who wrote, isnt good. Just mention what commands needed and how to use them... then you can add a link to original man page...

Back in smallbasic project, sourceforge had al ready did tickets and forums, but no, no one used (except 2-3 ppl). The reason it wasn;t attractive to visitors and users to involve, so I just add a phpBB forum, and wow was filled with ppl in 1-2 months commenting, asking and exchange scripts.

@the-maldridge:
I am sure there are plenty examples in srcpkgs... but someone has to point me instead of losing my time discovering a boring matter as a packager.
I have more important thinks to do, for example like patching the last grief version for linux, actually I did it yesterday for void-linux and I have problems to find, like what happend to /usr/include/ncurses/ and /usr/include/ncursesw.h... the previous time I was patched grief it was for slackware and slackware is very respecting to standard ways..

**Comment by flexibeast** [comment]
@nereusx:

> That is in your contributing: "There's a helper tool for committing packages in the xtools package: xbump"

Okay, thanks. (Although i wasn't the one that wrote CONTRIBUTING.)  i'll prepare a PR to have CONTRIBUTING elaborate on how to use `xbump`.

> About xbump "manual":
> xbump pkgname [git commit options] – Git commit a version bump
> Is it supposed to I understand it?

It certainly makes sense to me, though i admit English is my native language. Still, i agree it could do with more details - i'll see what i can do.

> Just mention what commands needed and how to use them.

Well, when i prepared my first package, i read CONTRIBUTING and found that it seemed to provide all the necessary information, as shown by the fact that the PR to which i referred in my previous comment did not require any further changes before it was merged.

> Back in smallbasic project, sourceforge had al ready did tickets and forums, but no, no one used (except 2-3 ppl). The reason it wasn;t attractive to visitors and users to involve, so I just add a phpBB forum, and wow was filled with ppl in 1-2 months commenting, asking and exchange scripts.

Okay, but there are a couple of things to keep in mind here:

* The FOSS world has changed significantly since Sourceforge was the main player in online code repos. Git is now the de facto standard DVCS in FOSS, and GitHub, even after its acquisition by Microsoft, is probably still the primary platform for project hosting and collaboration. (A number of people moved to GitLab, but in the last couple of weeks a number of people are probably considering moving elsewhere after the recent tracking and analytics debacle.) What this means is that *many* people nowadays are much more familiar with Git and GitHub than they are with things like CVS or SVN - enough so that projects not using Git or GitHub regularly get told that they 'should' move to using one or both, to make things easier for new contributors (examples from my experience: Emacs, OpenBSD), regardless of whether or not that makes things easier for core and/or regular contributors.

* Setting up and maintaining forum software requires extra time and effort from volunteers who can already have very full plates. (Again: Void is maintained entirely by volunteers.) Using Reddit as a general forum, and GitHub as an issue tracking forum, means that the Void devs have this work taken care of for them. Personally, i like the fact that the Void devs can spend their time improving the distro itself, rather than worrying about the latest security and functionality bugs in forum software.

> I am sure there are plenty examples in srcpkgs... but someone has to point me instead of losing my time discovering a boring matter as a packager.
> I have more important thinks to do

But why is your time as a volunteer inherently more valuable than the time of the volunteer Void devs?

When i packaged `hunspell-en_AU`, i thought it obvious that i should look at the `template` file for `hunspell-en_US` to give me an example of what to do. Still, it probably couldn't hurt to change the "Creating a new template" section of CONTRIBUTING to say something like:

> For deeper insights on the contents of template files consider reading the manual, and be sure to browse the existing template files in srcpkgs for concrete examples.

**Comment by nereusx** [comment]
Yes, github interface is simple and nice. I prefer it too. Microsoft sure did bad impact but still is nonsense to change server... Well I have a copy to my disks too, you know.

The meaning of all this is just make it simple and nice and ppl will come.

Your work to write tutorials/manuals its very important, and there are no many ppl that had the courage to do it. Try to keep ppl's interest and show them "oh but is too simple" is the key of success for me. Unfortunately in most cases I didn't hear myself and write very detailed docs :)

> But why is your time as a volunteer inherently more valuable than the time of the volunteer Void devs?

Going something wrong with C, is strong reason to change distro. I wont like to live the decade of 90s again!
I like void because has no systemd and it can continued updates with few problems (now has a mess with core-utils, net-utils and iputils); at least for me there are more important things than the distro itself.

**Pull request closed without merging**